test/tests/only_files_option.rb

```diff
@@ -19,7 +19,7 @@ def expected
   end
 
   def report
-    @@report ||= BrakemanTester.run_scan "rails3.2", "Rails 3.2", { :only_files => ["app/views/users/"], :skip_files => ["app/views/users/sanitized.html.erb"] }
+    @@report ||= BrakemanTester.run_scan "rails3.2", "Rails 3.2", { :only_files => ["app/views/users/", "exec_controller.rb", "/app/models/user/"], :skip_files => ["app/views/users/sanitized.html.erb"] }
```

Comment:
While this doesn't exercise 100% of the new logic, this seems to provide a good sense of things working as expected without dramatically refactoring this test suite. If you would like to see something else added just let me know.